### DISCUSSION TITLE
Limit File or Image Upload Size

### DISCUSSION THREAD

**Discussion by supermx1** [directus/directus#8686] [Feature Requests]
In the Directus app, while setting up a file or image field, we should be able to specify the maximum file size or image size (file size or *dimension). So it can be more enforceable from the frontend

**Comment by supermx1** [comment]
Hello

**Reply by rijkvanzanten** [reply]
Hi! 👋🏻

**Reply by supermx1** [reply]
Is this an idea that can work? I have tried using role and permissions to limit the filesize basically preventing the upload of an image greater than or equal to a set size in bytes, but that doesn't work. I wonder if it can be done on the field properties... Set a validation that prevents upload of large files or file types

**Reply by rijkvanzanten** [reply]
Absolutely! I'd love to have that configurable on the interface options, or even through permissions/validation by introspecting the selected file

**Reply by supermx1** [reply]
Perfect...

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
